package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "wmt-worker",
+  "version": "0.1.0",
+  "description": "WMT Async worker application",
+  "main": "start.js",
+  "repository": "https://github.com/ministryofjustice/wmt-worker.git",
+  "author": "Tom Swann",
+  "license": "MIT",
+  "scripts": {
+    "start": "node start.js",
```

Comment:
Think we're looking to move towards using gulp as the universal runner for tasks and that, leveraging it from npm as well:

  "main": "app/app.js",
  "scripts": {
    "start": "gulp",
    "test": "gulp test"
  },

Though, I was talking through this with andy and that was in regards to the mvc app so I'm not sure about here yet :[